src/components/measureLanding/MeasureLanding.tsx

```diff
@@ -108,6 +111,19 @@ export default function MeasureLanding() {
     }
   };
 
+  useEffect(() => {
+    if (featureFlags?.MeasureSearch) {
+      measureServiceApi.getMeasureCounts().then((data) => {
```

Comment:
I am guessing it is safe to add a catch block ?

Reply:
Added